### PR TITLE
Add Polish translation!

### PR DESCRIPTION
**Pull request by leavethevault**
A translation of the title, as you requested:

Interaktywny wstęp do transformaty Fouriera
Jen Swanson
Przekład: Jakub Górczyński

### CODE REVIEW

File: content/content-pl.md [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+Gdy się przyjrzysz, zauważysz, że właściwie tylko kilka pierwszych sinusoid ma największy wpływ na układ fali. Gdy suwak znajduje się w połowie zakresu, fala jest już mniej więcej ukształtowana, ale na jej powierzchni widnieją "wężyki". Fale znajdujące się w drugiej połowie zakresu suwaka są potrzebne, aby spłaszczyć "wężyki" i uzyskać gładką linię.
+
+Gdy posłuchasz fali, usłyszysz, że dźwięk się obniża, bo pozbyliśmy się wyższych częstotliwości.
+
+Ten proces zadziała dla każdej powtarzającej się linii. Daj się ponieść - narysuj swoją własną!
+
+<div class="multi-container">
+<div class="sketch" >
+    <canvas id="wave-draw" class="sketch-child" width=500 height=300></canvas>
+    <p id="wave-draw-instruction" class="instruction wave-instruction">Draw here!</p>
```
Comment (Jezzamonn) [inline content/content-pl.md]: This part is still in English

File: content/content-pl.md [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+*Użyj suwaka powyżej, aby zmienić ilość okręgów*
+
+Tak jak poprzednio, otrzymujemy całkiem niezłe przybliżenie naszego wzoru jedynie przy użyciu kilku okręgów. Ponieważ jest to dość prosty kształt, kolejne okręgi delikatnie wyostrzają krawędzie.
+
+Wszystko to można wykorzystać w jakimkolwiek rysunku. Naprawdę! Teraz masz okazję, żeby się tym pobawić.
+
+<div class="multi-container">
+<div class="sketch" >
+    <canvas id="draw-zone" class="sketch-child" width=500 height=500></canvas>
+    <p id="draw-zone-instruction" class="instruction">Draw here!</p>
```
Comment (Jezzamonn) [inline content/content-pl.md]: Draw here, and undo on the line below are in English still

**Comment by Jezzamonn** [comment]
There's one other "Draw Here!" left but I can fix that up using the other translation :)

**Comment by Jezzamonn** [comment]
Oh, some follow up questions actually:
I would put something like "Translated by ___" at the top. What should that phrase look like here? And do you that to link to anything?

**Comment by Jezzamonn** [comment]
Oh haha I just saw "Przekład: Jakub Górczyński". That works :) But would you like that to link to anything?

**Comment by leavethevault** [comment]
It could link to my github at github.com/leavethevault.
Sorry for missing the other "Draw Here!"


pt., 26 kwi 2019, 18:42 użytkownik Jez Swanson <notifications@github.com>
napisał:

**Comment by Jezzamonn** [comment]
Alright, just put in on the website, should be there once the cache clears!

**Comment by leavethevault** [comment]
The redirection to my GitHub throws 404, because the link looks like this: http://www.jezzamon.com/fourier/github.com/leavethevault

instead of this: https://github.com/leavethevault

**Comment by leavethevault** [comment]
Plus, 'polski' subpage title should start with the capital letter, just as 'English' or 'Deutsch'.

**Pull request merged**